packages/oas-to-har/src/lib/style-formatting/index.ts

```diff
@@ -225,7 +225,7 @@ function shouldExplode(parameter: ParameterObject) {
   );
 }
 
-export default function formatStyle(value: unknown, parameter: ParameterObject): any {
+export function formatStyle(value: unknown, parameter: ParameterObject): any {
```

Comment:
just checked and this one is fine, but we just wanna be careful that we're not making this change in any file that's exposed in [our `exports` field in the `package.json`](https://github.com/readmeio/oas/blob/4bccb292663cb3945062633cb48f66aa57a8dedd/packages/oas-to-har/package.json#L9) since that would be a breaking change (per what you mentioned [here](https://github.com/readmeio/oas/pull/992/files#r2280009596))

Reply:
yeah i definitely checked that before making any of these default changes i did